src/js/components/Carousel/carousel.stories.js

```diff
@@ -23,3 +23,7 @@ const SimpleCarousel = () => (
 );
 
 storiesOf('Carousel', module).add('Simple Carousel', () => <SimpleCarousel />);
+storiesOf('Carousel', module).add(
+  'Simple Carousel With `initialActiveIndex: 1`',
```

Comment:
We like to keep the names of the stories simple, so I'll be ok with just `Simple`, otherwise the name looks cumbersome.
![Screen Shot 2019-07-18 at 11 13 35 AM](https://user-images.githubusercontent.com/6320236/61477755-364b6f00-a94d-11e9-8d94-6987fad6c5f3.png)